packages/world/test.ts

```diff
@@ -35,55 +36,170 @@ describe("WorldReader", () => {
         const players = await reader.getPlayerData();
         expect(players).toHaveLength(1);
     });
-    test("#getRegionData", async () => {
-        const reader = await WorldReader.create(`${root}/saves/sample-map`);
-        const region = await reader.getRegionData(0, 0);
-        expect(region.DataVersion).toEqual(1976); // 1628
-        expect(region.Level.Entities).toBeInstanceOf(Array);
-        expect(region.Level.Sections).toBeInstanceOf(Array);
-    })
+
+    describe("#getRegionData", () => {
+        test("should be able to get region data", async () => {
+            const reader = await WorldReader.create(`${root}/saves/sample-map`);
+            const region = await reader.getRegionData(0, 0);
+            expect(region.DataVersion).toEqual(1976); // 1628
+            expect(region.Level.Entities).toBeInstanceOf(Array);
+            expect(region.Level.Sections).toBeInstanceOf(Array);
+        })
+        test("should be able to get the correct section", async () => {
+            let reader = await WorldReader.create(`${root}/saves/1.12.2`);
```

Comment:
Is this tested for the new format too?

Reply:
Actually, I'll add more test for querying the section. Currently only query for 0 section. I'll add a test for the section other than 0. I think the section query is the same between legacy and new version. Otherwise, the other test (seekblockStateId) won't be able get current tested.